src/NotifyProvider.tsx

```diff
@@ -36,19 +36,27 @@ export const NotifyProviderBase = ({ children }: PropsWithChildren<{}>) => {
       options,
       onPress,
       noTimeoutBar = false,
+      limit = 0,
     }: NotifyOptions) => {
-      setItems((prev) => [
-        ...prev,
-        {
-          id: generateUUID(10),
-          level,
-          message,
-          duration,
-          options,
-          onPress,
-          noTimeoutBar,
-        },
-      ]);
+      setItems((prev) => {
+        if (limit > 0) {
```

Comment:
Can you change initial value of the `limit` to null? 
When i saw this piece of the code, i thought "Why should limit greater than 0" and i checked twice the parameters. Its little bit confusing.

```js
//...
limit = null
//...

if (limit){ // if user provide limit. Better 👌
    // Do something 
}
```

Thanks

Reply:
Normally it shouldn't be `limit > 0`, obviously after writing the code I realized that I made this mistake. 😔